### DISCUSSION TITLE
Image Cropping action using cypress?

### DISCUSSION THREAD

**Discussion by mohinid** [cypress-io/cypress#18721] [Component Testing]
Can someone tell me how to perform Image Cropping using cypress?

**Comment by Arinzjos** [comment]
Hi, i am having this issue now, where you able to solve this? I will appreciate your help. Thanks

**Reply by mohinid** [reply]
Hi, to overcome this problem during testing, we had a workaround by adding a hidden skip button in that step and then handled the cropping from the backend. Could not use cypress for performing the action from the frontend.

**Reply by Arinzjos** [reply]
Ok thank you, i appreciate

**Comment by Parth909** [comment]
I was not able to perform dragging action for cropping the image

```js
cy.get("[data-test=image-to-crop]")
      .find("img")
      .should("be.visible")
      .trigger("click", 70, 71, {
        button: 1,
        force: true,
      })
      .trigger("mousedown", 70, 71, {
        button: 1,
        force: true,
      })
      .trigger("mousemove", 292, 197, {
        button: 1,
        force: true,
      })
      // .trigger("dragstart", 50, 50, {
      //   button: 1,
      //   force: true,
      // })
      // .trigger("dragend", 262, 197, {
      //   button: 1,
      //   force: true,
      // })
      .trigger("mouseup", { force: true })
      .wait(2000);
```
![image](https://user-images.githubusercontent.com/59220026/174026962-6c44b65c-b3f5-4618-aa57-1a114a51a46e.png)
This is what i wanted to do but 
But unable to do that using cypress

![image](https://user-images.githubusercontent.com/59220026/174027223-2efeb667-0a8d-4932-9f3e-0cff81dd6622.png)
